views/js/controller/runner/runner.js

```diff
@@ -81,9 +83,23 @@ define([
                     $('.runner').removeClass('hidden');
                 });
             })
+            .on('pause', function(data) {
+                if (data && data.reason) {
+                    // change exit Url
+                    params.reason = data.reason;                    
+                }
+            })
             .after('destroy', function () {
+                
                 // at the end, we are redirected to the exit URL
-                window.location = config.exitUrl;
+                var url = config.exitUrl;
+                if (params && params.reason) {
+                    url = new urlParser(url);
```

Comment:
You don't need the UrlParser but the [util/url](https://github.com/oat-sa/tao-core/blob/master/views/js/util/url.js#L150)